docker-compose.yml

```diff
@@ -23,7 +24,11 @@ services:
     image: mysql:5.7
     volumes:
       - ./db:/docker-entrypoint-initdb.d
-    restart: always
+    restart: on-failure
+    healthcheck:
```

Comment:
Nice! Haven't seen a good example of this before.

Reply:
healthcheck here just overrides whatever is defined in the container.
If you run `docker ps` or `docker-compose ps` you'll see "Up (Healthy)" instead of just up.

It's generally better to have an application specific healthcheck than this sort of trivial check, but this can be a hacky substitute for e.g. properly waiting for the DB to be available.